packages/rx_bloc/pubspec.yaml

```diff
@@ -1,6 +1,6 @@
 name: rx_bloc
 description: A Flutter package that helps implementing the BLoC (Business Logic Component) Design Pattern using the power of reactive streams.
-version: 5.0.0
+version: 5.1.0
```

Comment:
When upgrading dev dependencies we do not need to bump the version of the package

Reply:
Oversight. Noted for next time. ✌🏼